main.js

```diff
@@ -44,8 +45,10 @@ function resetForm() {
   deselectButton('exercise');
   deselectButton('meditate');
   deselectButton('study');
-}
-
+  hideElement(buttonError);
+  hideElement(minutesError);
+  hideElement(secondsError);
+};
```

Comment:
These additions ensure that error messages do not persist when the activity form is reset